Source/Core/VideoCommon/PostProcessing.cpp

```diff
@@ -137,8 +137,8 @@ void PostProcessingConfiguration::LoadOptions(const std::string& code)
           if (endpos != std::string::npos)
           {
             // New section!
-            std::string sub = line.substr(1, endpos - 1);
-            option_strings.push_back({sub});
+            std::string_view sub = line.substr(1, endpos - 1);
+            option_strings.push_back({std::string(sub)});
```

Comment:
`emplace_back` may be a better call here, not sure.

Reply:
It won't work. Keep in mind that `option_strings` is an `std::vector<GLSLStringOption>`, not an `std::vector<std::string>`.

Reply:
Fair enough.